src/components/SymbolModal.js

```diff
@@ -96,11 +96,13 @@ class SymbolModal extends Component {
   componentWillUnmount() {
     const shortcuts = this.context.shortcuts;
     shortcuts.off(L10N.getStr("symbolSearch.search.key2"));
+    shortcuts.off("Escape");
   }
 
   componentDidMount() {
     const shortcuts = this.context.shortcuts;
     shortcuts.on(L10N.getStr("symbolSearch.search.key2"), this.openSymbolModal);
+    shortcuts.on("Escape", this.closeModal);
```

Comment:
It was problematic adding shortcuts to the modal itself and would leave the shortcut disabled on its dismount. Letting the consumer's handle it seems like a simpler API.